catalog-rest-service/src/main/java/org/openmetadata/catalog/Entity.java

```diff
@@ -92,6 +92,9 @@
   public static final String USER = "user";
   public static final String TEAM = "team";
 
+  // Thread is just used in the tests
+  public static final String THREAD = "Thread";
```

Comment:
`thread` instead of `Thread` to follow same format?

Reply:
I have removed this constant since it will only be used in the test and also thread is not an entity.